WatchySim/WatchySim.cpp

```diff
@@ -299,13 +299,17 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT message,
 
         case ID_WIFI_ON:
             watchy.setWifi(true);
+            watchy.setWeatherExternal(true);
             InvalidateRect(hWnd, NULL, false);
             PostMessage(hWnd, WM_PAINT, 0, 0);
 
             return 0;
 
         case ID_WIFI_OFF:
             watchy.setWifi(false);
+            // Watchy SDK sets the weather code to 800 and flips the external flag when WiFi fails to connect
+            watchy.setWeatherCode(800);
```

Comment:
Will it report an ambient temperature? If so, when WiFi goes offline we should set the temperature to 20C.

Reply:
Good idea, currently it was just continuing to show the last temperature. My real Watchy was showing an internal temp of 27C when the WiFi dropped and it used the internal sensor, so I could set it to that, and I will make sure if you override the temp after that then turn on the WiFi it will stay at the temp selected.